app/views/practices/_search.js.erb

```diff
@@ -257,10 +242,10 @@ function searchPracticesPage() {
             // Print results to the page
             document.querySelector('#search-results').innerHTML = finalResults;
             // After the html for the card has been built and applied to the DOM, replace the image 'src' based on the signed url
-            replaceImagePlaceholders();
+            replaceSearchPageImagePlaceholders(practiceElements);
             $(SEARCH_RESULTS).showMoreItems({
-                startNum: 12,
-                afterNum: 12,
+                startNum: 24,
+                afterNum: 24,
```

Comment:
Interesting - what's the threshold change doing here?

Reply:
I had to add divider elements and those elements are added dynamically depending on the index of the search result above it, if the search result is the last one it doesn't add a divider

Reply:
@camillevilla I just realized that this wasn't exactly correct, what I needed to do was add the lines above the search result elements (when not the first) and set the start and after to 23, which is more in line with the design

Reply:
🤔 Hmm if we're needing to recalculate pagination due to adding `<hr>` elements, I'd suggest we implement those as a `border-bottom` via CSS instead. Added bonus of avoiding `<hr>` elements - we don't have to remember to add an `aria-hidden="true"` to them.